### PR TITLE
Fix not enough arguments

### LINKED ISSUE #7: not enough arguments
```
#20 125.1 /go/pkg/mod/github.com/mholt/caddy-ratelimit@v0.0.0-20210401181400-cdde3c722a44/distributed.go:106:87: not enough arguments in call to h.storage.Store
#20 125.1 	have (string, []byte)
#20 125.1 	want (context.Context, string, []byte)
#20 125.1 /go/pkg/mod/github.com/mholt/caddy-ratelimit@v0.0.0-20210401181400-cdde3c722a44/distributed.go:116:54: not enough arguments in call to h.storage.List
#20 125.1 	have (string, bool)
#20 125.1 	want (context.Context, string, bool)
#20 125.1 /go/pkg/mod/github.com/mholt/caddy-ratelimit@v0.0.0-20210401181400-cdde3c722a44/distributed.go:132:34: not enough arguments in call to h.storage.Load
#20 125.1 	have (string)
#20 125.1 	want (context.Context, string)
```

### PR DESCRIPTION
**Pull request by sylloger**
Close #7

### CODE REVIEW

**Comment by francislavoie** [comment]
Ah, yeah. That's due to https://github.com/caddyserver/certmagic/pull/155, so this plugin stopped being compatible with the latest version of Caddy.

Thanks for the fix!

**Comment by francislavoie** [comment]
I think the `go.mod` should be updated as well, probably, since it now targets a newer version of certmagic and Caddy by making these changes.

**Comment by mholt** [comment]
That's weird. 🤔  I believe go.mod will also need to be updated to the latest versions of CertMagic / Caddy, as currently this change is not compatible with the pinned versions.

**Comment by sylloger** [comment]
@francislavoie @mholt Thank you for bringing this to my attention. I ran into the error while compiling in xcaddy, I didn't realize I needed to update `go.mod` at the time, I've since updated it.

Please let me know if you require any other changes.

**Review by mholt — APPROVED** [review]
Great, thank you!

File: go.mod [hunk 1]
```diff
@@ -1,9 +1,112 @@
...
+	golang.org/x/xerrors v0.0.0-20220411194840-2f41105eb62f // indirect
+	google.golang.org/genproto v0.0.0-20220222213610-43724f9ea8cf // indirect
+	google.golang.org/grpc v1.44.0 // indirect
+	google.golang.org/protobuf v1.28.0 // indirect
+	gopkg.in/square/go-jose.v2 v2.6.0 // indirect
+	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
+	howett.net/plist v1.0.0 // indirect
+)
+
+replace github.com/lucas-clemente/quic-go => github.com/lucas-clemente/quic-go v0.26.0
```
Comment (mholt) [inline go.mod:112]: Wow, I admit I did not see this 👀

**Pull request merged**